Docs: refactor "skipping a build" section (#9717)
readthedocs/readthedocs.org: docs/user/build-customization.rst

         - git fetch --unshallow

Cancel build based on a condition
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When a command exits with code ``439``,

Reviewer: This error code isn't a valid exit code (https://tldp.org/LDP/abs/html/exitcodes.html), we shouldn't document a code above 255 otherwise users will get confused.

Read the Docs will cancel the build immediately.

> We can use this approach to cancel builds we don't want to complete based on a condition.

Reviewer: Who is `We`? I think we want to talk to the user right?

```suggestion
You can use this approach to cancel builds that you don't want to complete based on some conditional logic.
```

Author: Yes, "we" is incorrect here.
